### ISSUE TITLE
Package phine/phar is abandoned, use box-project/box2 instead.

### ISSUE THREAD

**Issue by carbontwelve** [carbontwelve/tapestry#96]
```
Package phine/exception is abandoned, you should avoid using it. No replacement was suggested.
Package phine/path is abandoned, you should avoid using it. No replacement was suggested.
Package phine/observer is abandoned, you should avoid using it. No replacement was suggested.
Package phine/phar is abandoned, you should avoid using it. Use box-project/box2 instead.
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.
```

**Comment by carbontwelve** [comment]
Possibly in the same way as https://github.com/codeclimate/php-test-reporter ??

**Comment by carbontwelve** [comment]
This is fixed by 2e8117f4db362f2f090ac3bbf9852e8aeea3bfec
